src/container_support/utils.py

```diff
@@ -30,7 +31,7 @@ def download_s3_resource(source, target):
     """ Downloads the s3 object source and stores in a new file with path target.
     """
     print("Downloading {} to {}".format(source, target))
-    s3 = boto3.resource('s3')
+    s3 = boto3.resource('s3', region_name=os.environ['AWS_REGION'])
```

Comment:
I think a os.environ.get() would be better. Since, this will fail if the container doesn't have this environment variable defined.